### PR TITLE
Homework 20

### PR DESCRIPTION
**Pull request by franzferdinnand**

### CODE REVIEW

File: Lesson_20/test_functions.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        self.fibo(1)
+        self.assertEqual(len(self.fibo.cache), 2)
+
+    def test_nothing_in_args(self):
+        with self.assertRaises(TypeError) as error:
+            self.fibo()
+
+
+class FormattedNameTestCase(TestCase):
+    def test_positive_case(self):
```
Comment (mikolaz27) [inline Lesson_20/test_functions.py:47]: The test cases should be named more descriptively to make it easier to understand what each test case is checking. For example, the test_positive_case could be renamed to test_formatted_name_with_three_arguments.

File: Lesson_20/test_functions.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        self.assertEqual(formatted_name("Serhii", "Turchyn", "Viktor"), "Serhii Viktor Turchyn")
+
+    def test_format_title_works_case(self):
+        self.assertEqual(formatted_name("serhii", "turchyn", "viktor"), "Serhii Viktor Turchyn")
+
+    def test_two_names_case(self):
+        self.assertEqual(formatted_name("Serhii", "Turchyn"), "Serhii Turchyn")
+
```
Comment (mikolaz27) [inline Lesson_20/test_functions.py:55]: The test cases could include more edge cases, such as when the input strings contain special characters, leading or trailing spaces, or non-alphabetical characters.
